parse_race_lap_chart.py

```diff
@@ -90,7 +90,7 @@ def to_json(df: pd.DataFrame):
     session_type = 'R'
 
     # Convert to json
-    df['lap'] = df.apply(lambda x: Lap(number=x['lap'], position=x['position']), axis=1)
+    df['lap'] = df.apply(lambda x: Lap(lap_number=x['lap'], position=x['position'], time=0.0), axis=1)
```

Comment:
I will probably change the entire race lap data stuff later, as the current parsing really mess a lot of things...